src/migrations/20201019203657-add-businessId-to-user.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('user', 'businessId', {
```

Comment:
This requires you to create the business **first**, then create a user associating it to the business you just created. I don't think that's a business logic you want to have in the backend.

Reply:
Remove this migration, you already created an association table.